core/domain/exp_domain.py

```diff
@@ -99,6 +99,9 @@
 # Represents answers which were predicted using a statistical training model
 # from training data within an answer group.
 STATISTICAL_CLASSIFICATION = 'statistical_classifier'
+# Represents answers which were predicted using a machine learning model from
+# training data within an answer group.
+MACHINE_LEARNING_CLASSIFICATION = 'ml_classifier'
```

Comment:
Don't define a new thing here; this is the same as statistical_classification so please use that instead.